package-lock.json

```diff
@@ -7,10 +7,13 @@
     "": {
       "name": "scratch-gui",
       "version": "1.8.107",
+      "hasInstallScript": true,
```

Comment:
Since this `hasInstallScript` line is adjacent to the version number, which is repeatedly changing in the `develop` branch, this PR will report merge conflicts most of the time. If this PR is approved, I'll resolve the conflict one last time before merging the PR.